platform/ios/docs/guides/For Style Authors.md

```diff
@@ -169,6 +169,24 @@ To create a shape source from local GeoJSON data, first
 [convert the GeoJSON data into a shape](working-with-geojson-data.html#converting-geojson-data-into-shape-objects),
 then use the `-[MGLShapeSource initWithIdentifier:shape:options:]` method.
 
+# Style functions and interpolation mode
+
+The runtime styling API introduced `MGLStyleFunction` to the iOS SDK.
+[Data-driven styling](data-driven-styling.html) expands `MGLStyleFunction` and
+introduces `MGLInterpolationMode`. Individual style property documentation
+includes which subclasses of `MGLStyleFunction` are enabled for that property.
+
+In style JSON                | In the SDK
+-----------------------------|-----------
+`zoom function`              | `MGLCameraStyleFunction`
+`property function`          | `MGLSourceStyleFunction`
+`zoom-and-property functions`| `MGLCompositeStyleFunction`
+`type`                       | `MGLInterpolationMode`
```

Comment:
This is also a terminological difference. It’s worth mentioning that a “function type” is an “interpolation mode” up in the first table.